bundles/org.openhab.binding.nikohomecontrol/README.md

```diff
@@ -161,106 +153,59 @@ Note down the `actionId` parameter from the thing, remove it before adding it ag
+The same applies applies for `thermostatId`, `meterId` and `accessId`.
 
-The `step` parameter is only available for dimmers.
-It sets a step value for dimmer increase/decrease actions.
-The parameter is optional and set to 10 by default.
-
-The Thing configuration for **Niko Home Control thermostats** has the following syntax:
+An example **action** textual configuration looks like:
 
 ```java
-Thing nikohomecontrol:thermostat:<bridgeId>:<thingId> "Label" @ "Location"
-                        [ thermostatId="<Niko Home Control thermostat ID>",
-                          overruleTime=<default duration for overrule temperature in minutes> ]
+Thing nikohomecontrol:dimmer:mybridge:mydimmer [ actionId="1", step=5 ]
 ```
 
-or nested in the bridge configuration:
+For **thermostats** it would be:
 
 ```java
-thermostat <thingId> "Label" @ "Location" [ thermostatId="<Niko Home Control thermostat ID>" ]
+Thing nikohomecontrol:thermostat:mybridge:mythermostat [ thermostatId="abcdef01-abcd-1234-ab98-012345abcdef", overruleTime=10 ]
 ```
 
-`thingId` can have any value, but will be set to the same value as the thermostatId parameter if discovery is used.
-
-`"Label"` is an optional label for the Thing.
-
-`@ "Location"` is optional, and represents the location of the thing.
-Auto-discovery would have assigned a value automatically.
-
-The `thermostatId` parameter is the unique IP Interface Object ID as automatically assigned in the Niko Home Control I Controller when programming the Niko Home Control system using the Niko Home Control programming software.
-It is not directly visible in the Niko Home Control programming or user software, but will be detected and automatically set by openHAB discovery.
-For textual configuration, it can be retrieved from the .nhcp configuration file.
-
-For Niko Home Control II, the `thermostatId` parameter is a unique ID for the thermostat in the controller.
-It can only be auto-discovered.
-If you want to define the thermostat through textual configuration, you may first need to do discovery on the bridge to get the correct `thermostatId` to use in the textual configuration.
-
-The `overruleTime` parameter is used to set the standard overrule duration in minutes when you set a new setpoint without providing an overrule duration.
-The default value is 60 minutes.
-
-The Thing configuration for **Niko Home Control energy meters** has the following syntax:
+For **energy meters**:
 
 ```java
-Thing nikohomecontrol:energymeter:<bridgeId>:<thingId> "Label" @ "Location"
-                        [ energyMeterId="<Niko Home Control energy meter ID>" ]
+Thing nikohomecontrol:energymeter:mybridge:mymeter [ meterId="3" ]
 ```
 
-or nested in the bridge configuration:
+For **access devices**:
 
 ```java
-energymeter <thingId> "Label" @ "Location" [ energyMeterId="<Niko Home Control energy meter ID>" ]
+Thing nikohomecontrol:accessRingAndComeIn:mybridge:myaccess [ accessId="abcdef01-dcba-1234-ab98-012345abcdef", refresh=30 ]
```

Comment:
refresh parameter is not mentioned as applicable for the accessRingAndComeIn thing type in the table above.